executor/modes/local/runner.py

```
@@ -314,6 +331,36 @@ async def close_task_session(self, task_id: int) -> None:
+        """
+        task_ids = list(self._running_tasks.keys())
+        if not task_ids:
+            logger.info("[Runner] No tasks to cancel")
+            return
+
+        logger.info(f"[Runner] Cancelling {len(task_ids)} running task(s)")
+        for task_id in task_ids:
+            try:
+                await self.cancel_task(task_id)
+                logger.info(f"[Runner] Cancelled task {task_id}")
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)